omc3/optics_measurements/rdt.py

```diff
@@ -179,21 +196,28 @@ def add_freq_to_header(header, plane, rdt):
     return mod_header
 
 
-def _process_rdt(meas_input, input_files, phase_data, invariants, plane, rdt):
+def _process_rdt(meas_input: DotDict, input_files: InputFiles, phase_data, invariants, plane, rdt: RDTTuple):
+    # Todo: only on-momentum required? If not, remove this or set `dpp_value=None`, see #456
+    dpp_value = 0  
+    invariants = {plane: inv[input_files.dpp_frames_indices(plane, dpp_value)] for plane, inv in invariants.items()}
```

Comment:
Same as in a previous file, I'd appreciate a comment here explaining what "filtering" we are doing here. We will have forgotten soon enough